src/WebJobs.Script/Rpc/LanguageWorkerChannelFactory.cs

```diff
@@ -38,7 +38,7 @@ public ILanguageWorkerChannel CreateLanguageWorkerChannel(string scriptRootPath,
                 throw new InvalidOperationException($"WorkerCofig for runtime: {runtime} not found");
             }
             string workerId = Guid.NewGuid().ToString();
-            ILogger workerLogger = _loggerFactory.CreateLogger($"LanguageWorkerChannel.{runtime}.{workerId}");
+            ILogger workerLogger = _loggerFactory.CreateLogger($"Worker.LanguageWorkerChannel.{runtime}.{workerId}");
```

Comment:
Ideally we'd have a method to construct the category -- like we have in SDK here: https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogCategories.cs#L55.

But this works for now...